src/kyc_api.ts

```diff
@@ -0,0 +1,43 @@
+import axios, { AxiosInstance } from 'axios'
+
+export interface AccessToken {
+    token: string
+    userId: string
+}
+
+const BASE_URL = 'http://localhost:4000/api/v1'
```

Comment:
We have a network configuration file (src\store\modules\network\network.ts) where we usually store all of our URL's.
This network struct already contains the URL to the node API, and, we should not rely on deprecated v1.

Or what is exactly listening on 4000 (node?? or sumsub??)

Reply:
Ignore the port 4000 and localhost. We will basically point to the API written by @11Yupi11 to communicate with sumsub

Reply:
So is it right that KYC state is requested using sumsub server and not node?
Maybe it would be better to directly request on api node from smart contract, to avoid centralized services.

What do you think?

Reply:
KYC state is taken from a smart contract that correspond with the state of KYC in SumSub. We do not communicate with the node

Reply:
Question is if it would be better to change it to a direct communication with the node.
Wallet is already requesting information from a node, so why connect to a single point of failure service ??

Reply:
Do you mean a direct communication with a smart contract? That's doable. It's only for fetching the state of KYC. I don't grasp the idea of _requesting information from a node_. What information?

Reply:
@c4t-ag Are you interesting why we created additional service for this and not use the node?....if this is the question then we can answer that Sumsub can communicate only by https:// and also we can't store token and secret key in opensource repository as they might be secure.